src/Umbraco.Tests/Persistence/Repositories/LanguageRepositoryTest.cs

```diff
@@ -153,7 +152,7 @@ public void Can_Perform_GetByQuery_On_LanguageRepository()
                 // Assert
                 Assert.That(result, Is.Not.Null);
                 Assert.That(result.Any(), Is.True);
-                Assert.That(result.FirstOrDefault().CultureName, Is.EqualTo("da-DK"));
+                Assert.That(result.FirstOrDefault().IsoCode, Is.EqualTo("da-DK"));
```

Comment:
We actually want to ensure the `IsoCode` is correct (previously the `CultureName` was coincidentally the same).